tests/scripts/update_metadata.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Determines the size of generated files and updates check_metadata.sh to
+# reflect the new values.
+
+blockly_size=$(wc -c < "blockly_compressed.js")
+blocks_size=$(wc -c < "blocks_compressed.js")
+blockly_gz_size=$(wc -c < "blockly_compressed.js.gz")
+blocks_gz_size=$(wc -c < "blocks_compressed.js.gz")
```

Comment:
One note: you are measuring the sizes of the checked-in build products, rather than rebuilding from scratch.  That's fine (at least until we stop checking in build products), but it might be worth making that clear in appropriate places to avoid confusion down the line.

(Alternatively, since you are installing `node.js` anyway, you could `npm run build:compressed` and then measure the sizes of the copies in `build/` instead.)

Reply:
Updated to build and calculate the size of the new artifacts.